src/resources/LNPay.ts

```diff
@@ -0,0 +1,3 @@
+export const LNPay = {
+  DeveloperDashboardUrl: 'https://lnpay.co/developers/dashboard'
```

Comment:
I think this should go in the URLs file. I believe I already have another LNPay url there